plugins/android-extensions/android-extensions-compiler/testData/parcel/box/customSerializerWriteWith.kt

```diff
@@ -1,3 +1,5 @@
+// IGNORE_BACKEND: JVM
+// The JVM backend is missing support for custom parcelers in List<String>
```

Comment:
It'd be really nice to mention the specific YouTrack issues you've created in these ignore directives

Reply:
Good idea. Done!